srcpkgs/deepin-editor/template

```diff
@@ -14,7 +14,7 @@ license="GPL-3.0-or-later"
 homepage="https://github.com/linuxdeepin/deepin-editor"
 changelog="https://github.com/linuxdeepin/deepin-editor/blob/${version}/CHANGELOG.md"
```

Comment:
`changelog="https://raw.githubusercontent.com/linuxdeepin/deepin-editor/master/CHANGELOG.md"`

Reply:
There is no consensus on whether to provide a raw or browser link changelog.

Reply:
it's okay.